### PR TITLE
[all] Add computeBbox function to two forcefields

### PR DESCRIPTION
**Pull request by untereiner**
______________________________________________________

This PR: 
- [x] builds with SUCCESS for all platforms on the CI.
- [x] does not generate new warnings.
- [x] does not generate new unit test failures.
- [x] does not generate new scene test failures.
- [x] does not break API compatibility.
- [x] is more than 1 week old (or has fast-merge label).

**Reviewers will merge only if all these checks are true.**

### CODE REVIEW

**Comment by untereiner** [comment]
[ci-build]

File: modules/SofaGeneralSimpleFem/TetrahedralCorotationalFEMForceField.inl [hunk 1]
```diff
@@ -1304,6 +1304,30 @@ void TetrahedralCorotationalFEMForceField<DataTypes>::applyStiffnessPolar( Vecto
...
 
+template<class DataTypes>
+void TetrahedralCorotationalFEMForceField<DataTypes>::computeBBox(const core::ExecParams* params, bool onlyVisible)
+{
+	if( !onlyVisible ) return;
+
+	helper::ReadAccessor<DataVecCoord> x = this->mstate->read(core::VecCoordId::position());
+
+	static const Real max_real = std::numeric_limits<Real>::max();
+	static const Real min_real = std::numeric_limits<Real>::min();
```
Comment (fredroy) [inline modules/SofaGeneralSimpleFem/TetrahedralCorotationalFEMForceField.inl]: Warning : std::numeric_limits<Real>::min() returns 0 ! (not the lowest value of a Real)
You should use std::numeric_limits<Real>::lowest()

File: SofaKernel/modules/SofaSimpleFem/TetrahedronFEMForceField.inl [hunk 2]
```diff
@@ -1848,6 +1848,29 @@ inline void TetrahedronFEMForceField<DataTypes>::addDForce(const core::Mechanica
...
 
+template<class DataTypes>
+void TetrahedronFEMForceField<DataTypes>::computeBBox(const core::ExecParams* params, bool onlyVisible)
+{
+	if( !onlyVisible ) return;
+
+	helper::ReadAccessor<DataVecCoord> x = this->mstate->read(core::VecCoordId::position());
+
+	static const Real max_real = std::numeric_limits<Real>::max();
+	static const Real min_real = std::numeric_limits<Real>::min();
```
Comment (fredroy) [inline SofaKernel/modules/SofaSimpleFem/TetrahedronFEMForceField.inl]: (bis)

**Comment by damienmarchal** [comment]
[ci-build][with-scene-tests]
  
Comme ça.

File: modules/SofaGeneralSimpleFem/TetrahedralCorotationalFEMForceField.inl (hunk 1 above)
Comment (untereiner) [reply modules/SofaGeneralSimpleFem/TetrahedralCorotationalFEMForceField.inl]: shit ok

**Comment by untereiner** [comment]
[ci-build][with-scene-tests]

**Comment by damienmarchal** [comment]
[ci-build]

**Comment by untereiner** [comment]
[ci-reformat]

**Comment by untereiner** [comment]
[ci-build][with-scene-tests][ci-reformat]

**Comment by guparan** [comment]
The formatter was not ready yet. It should be now :-)
[ci-reformat]

**Comment by guparan** [comment]
Well well well, it seems it is not ready at all... I'm gonna clean this mess.

**Comment by guparan** [comment]
Let's try again [ci-reformat]

**Pull request merged**